_headers

```diff
@@ -11,3 +11,7 @@
   cache-control: public,max-age=31536000,immutable
 /favicon.svg
   cache-control: public,max-age=3600
+
+# [custom headers]
```

Comment:
Let's call this `[csp headers]` since that is what it is.

Reply:
Ok. Done